src/rnn.c

```diff
@@ -76,33 +76,39 @@ static OPUS_INLINE float relu(float x)
    return x < 0 ? 0 : x;
 }
 
+void faxpy(float *restrict a, const rnn_weight *restrict b, int k, float u)
```

Comment:
Since this function is not used outside of this translation unit:
```suggestion
static void faxpy(float *restrict a, const rnn_weight *restrict b, int k, float u)
```